application/tests/unit/common/models/EventLogTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+namespace tests\unit\common\models;
+
+use common\models\EventLog;
+use common\models\User;
+use yii\codeception\DbTestCase;
+
+use tests\unit\fixtures\common\models\EventLogFixture;
+use tests\unit\fixtures\common\models\MethodFixture;
+use tests\unit\fixtures\common\models\UserFixture;
+
+/**
+ * Class EventLogTest
+ * @package tests\unit\common\models
+ * @method User users($key)
+ * @method EventLog eventLogs($key)
+ */
+class EventLogTest extends DbTestCase
+{
+    public function fixtures()
+    {
+        return [
+            'users' => UserFixture::className(),
+            'event_logs' => EventLogFixture::className(),
+        ];
+    }
+
+    public function testLogNoUser()
+    {
+        EventLog::log('test-1461183312', 'testing-1461183312');
+
+        /*
+         * Check if entry was created
+         */
+        $eventLog = EventLog::findOne(['topic' => 'test-1461183312']);
+
+        $this->assertNotNull($eventLog);
+    }
+
+    public function testLogInvalidUser()
+    {
+        $this->setExpectedException('\Exception', '', 1461182172);
+        EventLog::log('test-1461183456', 'testing-1461183456', 44);
```

Comment:
If our list of user fixtures ever reaches the point of having 44 users in it, would this test then fail?